frontend/src/metabase/nav/components/search/SearchResults/SearchResults.tsx

```diff
@@ -75,11 +87,12 @@ export const SearchResults = ({
   }, [searchText, reset]);
 
   const hasResults = list.length > 0;
+  const showFooter = hasResults && footer && metadata;
 
   if (isLoading) {
     return (
       <Stack p="xl" align="center">
-        <Loader size="lg" />
+        <Loader size="lg" data-testid="loading-spinner" />
```

Comment:
I'll refactor `LoadingSpinner` in a separate PR to use Mantine components so we don't have a bunch of these `Stack > Loader[data-testid="loading-spinner"]` elements in our code